### PR TITLE
Add repo files.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Contributing In General
+
+Our project welcomes external contributions! If you have an itch, please
+feel free to scratch it.
+
+To contribute code or documentation, please submit a pull request to the [GitHub
+repository](https://github.com/IBM/speech-store).
```
Comment (stevemart) [inline CONTRIBUTING.md]: vr-speech-sandbox-cardboard

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+A good way to familiarize yourself with the codebase and contribution process is
+to look for and tackle low-hanging fruit in the [issue
+tracker](https://github.com/IBM/speech-sandbox/issues). Before embarking on
```
Comment (stevemart) [inline CONTRIBUTING.md]: vr-speech-sandbox-cardboard

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+with us.
+
+**We appreciate your effort, and want to avoid a situation where a contribution
+requires extensive rework (by you or by us), sits in the queue for a long time,
+or cannot be accepted at all!**
+
+### Proposing new features
+
+If you would like to implement a new feature, please [raise an
+issue](https://github.com/IBM/speech-store/issues) before sending a pull
```
Comment (stevemart) [inline CONTRIBUTING.md]: vr-speech-sandbox-cardboard

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+request so the feature can be discussed. This is to avoid you spending your
+valuable time working on a feature that the project developers are not willing
+to accept into the code base.
+
+### Fixing bugs
+
+If you would like to fix a bug, please [raise an
+issue](https://github.com/IBM/speech-store/issues) before sending a pull
```
Comment (stevemart) [inline CONTRIBUTING.md]: vr-speech-sandbox-cardboard

File: DEVELOPING.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Project Structure
+
+Here is a general overview of some of the more important directories in the project.
+
+```
+.
+├── data
```
Comment (stevemart) [inline DEVELOPING.md]: not sure if this is updated...

Comment (scottdangelo) [inline DEVELOPING.md]: We can leave this in the Developing.md
I need to add this and put workspace.json into it. I'll do that next patch.

**Pull request merged**